osu.Framework.Android/AndroidGameView.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu-framework/master/LICENCE
+
+using System;
+using Android.Content;
+using Android.Runtime;
+using Android.Util;
+using osuTK.Graphics;
+
+namespace osu.Framework.Android
+{
+    public abstract class AndroidGameView : osuTK.Android.AndroidGameView
+    {
+        int viewportWidth, viewportHeight;
```

Comment:
Add private modifier

```suggestion
        private int viewportWidth, viewportHeight;
```